src/commandDetails/coin/privacy.ts

```diff
@@ -0,0 +1,64 @@
+import { container } from '@sapphire/framework';
+import { Permissions, User } from 'discord.js';
+import {
+  CodeyCommandDetails,
+  CodeyCommandOptionType,
+  CodeyCommandResponseType,
+  SapphireMessageExecuteType
+} from '../../codeyCommand';
+import { getUserPrivacy, changeUserPrivacy } from '../../components/coin';
+
+// Update coin leaderboard privacy of a user
+const coinUpdateExecuteCommand: SapphireMessageExecuteType = async (client, messageFromUser, args) => {
```

Comment:
Change this to coinPrivacyExecuteCommand

Reply:
Fixed